crates/aptos-rest-client/src/error.rs

```diff
@@ -151,14 +151,14 @@ pub enum RestError {
     Bcs(bcs::Error),
     #[error("JSON er/de error {0}")]
     Json(serde_json::Error),
-    #[error("Web client error {0}")]
-    WebClient(reqwest::Error),
     #[error("URL Parse error {0}")]
     UrlParse(url::ParseError),
     #[error("Timeout waiting for transaction {0}")]
     Timeout(&'static str),
     #[error("Unknown error {0}")]
     Unknown(anyhow::Error),
+    #[error("Http error {0}")]
+    Http(u16),
```

Comment:
Could we use a hyper::StatusCode instead here, to make it clearer? Currently you can't tell it's a HTTP status without looking elsewhere.